93 streamline ui (#183)
datasciencecampus/pyGrams: README.md

python pygrams.py -p='avg'
```

#### Normalise by document length (-ndl)

> This option normalises the TYF-IDF cores by document length.

Reviewer: ```suggestion
This option normalises the TF-IDF scores by document length.
```

The first file (stopwords_glob.txt) contains stopwords that are applied to all n-grams.
The second file contains stopwords that are applied to all n-grams for n > 1 (bigrams and trigrams) and the last file (stopwords_uni.txt) contains stopwords that apply only to unigrams. The users can append stopwords into this files, to stop undesirable output terms.

### Folder structure

> Explain the folder structure

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```
